compair/static/less/assignment.less

```diff
@@ -123,12 +123,12 @@
       border-bottom: 1px solid @border-gray;
 
       /* limit how much of answer/comment content is shown */
-      > .content {
+      [content="answer.content"] .content {
```

Comment:
maybe we can add a new class to the two `<div class="{{answer.id}}" get-height>` tags (ex: `comparable`) so we can use `.comparable .content {` instead of using `[content="answer.content"] .content {`. 

Then we can also get rid of the `> .instructor-answer .content {...}` css and remove the `<div class="instructor-answer">` tag from `assignment-view-partial.html` since it is only used for this.